x-pack/plugins/lens/public/datatable_visualization/expression.tsx

```diff
@@ -161,6 +231,58 @@ export const getDatatableRenderer = (dependencies: {
   },
 });
 
+function possiblyWrapFormat(rowFormatted: string, type: string | undefined) {
+  if (type !== 'ip') {
+    return rowFormatted;
+  }
+  // Note this supports only IPv4: maybe there already a solution to support IPv6 and more?
```

Comment:
Ip fields can also store ipv6 values in Elasticsearch: https://www.elastic.co/guide/en/elasticsearch/reference/current/ip.html

To me it feels weird to just support ip4, but always attempt to sort. IMHO we should split that feature out and disable sorting on ip columns for now and tackle that separately.

Reply:
Yes, agree. There are also special values we may want to handle (`localhost` as/or `127.0.0.1`?).